project_templates/technote_spietex/{{cookiecutter.series.lower()}}-{{cookiecutter.serial_number}}/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/{{ cookiecutter.series.lower() }}--{{ cookiecutter.serial_number }}-lsst.io-brightgreen.svg
+   :target: https://{{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}.lsst.io
+.. image:: https://travis-ci.com/{{ cookiecutter.github_org }}/{{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}.svg
+   :target: https://travis-ci.com/{{ cookiecutter.github_org }}/{{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}
+
+{{ "#" * cookiecutter.title|length }}
+{{ cookiecutter.title }}
+{{ "#" * cookiecutter.title|length }}
+
+{{ cookiecutter.series.upper() }}-{{ cookiecutter.serial_number }}
+{{ "=" * (cookiecutter.series|length + cookiecutter.serial_number|length + 1) }}
+
+{{ cookiecutter.abstract }}
+
+Links
+=====
+
+- Live drafts: https://{{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}.lsst.io
+- GitHub: https://github.com/{{ cookiecutter.github_org }}/{{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}
+
+Build
+=====
+
+This repository includes lsst-texmf_ as a Git submodule.
+Clone this repository::
+
+    git clone --recurse-submodules https://github.com/{{ cookiecutter.github_org }}/{{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}
+
+Compile the PDF::
+
+    make
+
+Clean built files::
+
+    make clean
+
+Updating acronyms
```

Comment:
SPIE and ADASS are not going to be generating acronyms. Authors need to define terms on first usage.

Reply:
yes they are removed from the generated file but in the makefile.